### ISSUE TITLE
Testimonial Delay

### ISSUE THREAD

**Issue by dhruvabhat24** [EddieHubCommunity/BioDrop#10147]
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

Testimonial Section 

### Version of BioDrop (for example "v1.2.3")

v2.99.4

### Description

I have received one testimonial a day before even that is closed #10144 
But till now it is not displayed in my profile 

### Steps to Reproduce

Nil

### Do you want to work on this issue?

No

**Comment by ChinmayMhatre** [comment]
@dhruvabhat24 I believe it is visible now. But I'll open the repo for conversation to check if this delay is happening to more people. We can have a discussion on this issue is persists!

**Comment by dhruvabhat24** [comment]
Yes now it is visible

**Comment by eddiejaoude** [comment]
A new issue can be raised if it happens again, and we can then prioritise it again - but it seemed like it was a one off 🤞

**Comment by eddiejaoude** [comment]
No problem, but it has only been 1 hour since the testimonial - please can we give it time to see if the standard workflow works

**Comment by dhruvabhat24** [comment]
@eddiejaoude  okay

**Comment by dhruvabhat24** [comment]
It's been more than 48 hours and it didn't fetch

**Comment by eddiejaoude** [comment]
Yes, testimonials are reloaded when the app is deployed, you can see the code here https://github.com/EddieHubCommunity/BioDrop/blob/main/.github/workflows/vercel.yml#L55

From the deploy GitHub Action there has been no deployments for 3 days https://github.com/EddieHubCommunity/BioDrop/actions/workflows/vercel.yml

I think the code to load the testimonials should be moved from the deploy app to the testimonials GH Action 
https://github.com/EddieHubCommunity/BioDrop/blob/9ed1736d560cbf7ef5c95bc5f94a1fd726a03b2e/.github/workflows/vercel.yml#L55

**Comment by eddiejaoude** [comment]
I have manually reloaded testimonials, please check your account now

**Comment by dhruvabhat24** [comment]
I have checked now but still it is displayed on account I think you need to make the testimonial sections to GitHub actions

**Comment by ArvindParekh** [comment]
Hi! I'd like to help with this. Although I do not know about GH Actions (yet), I'm willing to learn on the go. I read about it and here's what I think needs to be done:

- At present, the code to reload the testimonials lies in the [vercel.yml](https://github.com/EddieHubCommunity/BioDrop/blob/main/.github/workflows/vercel.yml) file, where testimonials are only reloaded when the `load-data` job runs, which depends on the `deploy` workflow. 
- So I should figure out a way to update the testimonials without needing the deploy job to run.

Am I correct? Also, do I have to make changes to the [testimonial.yml](https://github.com/EddieHubCommunity/BioDrop/blob/main/.github/workflows/testimonial.yml) or just to [vercel.yml](https://github.com/EddieHubCommunity/BioDrop/blob/main/.github/workflows/vercel.yml)?
